app/src/main/java/edu/uc/jonesbr/myplantdiary/service/PlantService.kt

```diff
@@ -10,7 +10,7 @@ import retrofit2.Response
 
 class PlantService {
 
-    fun fetchPlants(plantName: String) : MutableLiveData<ArrayList<Plant>> {
+    internal fun fetchPlants(plantName: String) : MutableLiveData<ArrayList<Plant>> {
```

Comment:
Updated the visibility modifiers per the suggestions here: https://kotlinlang.org/docs/reference/visibility-modifiers.html